### ISSUE TITLE
test arraydiff error

### ISSUE THREAD

**Issue by cdeil** [astropy/regions#79]
@astrofrog - I updated to `regions` master just now and got this error:
```
$ python setup.py test -V
/opt/local/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/Cython/Distutils/old_build_ext.py:30: UserWarning: Cython.Distutils.old_build_ext does not properly handle dependencies and is deprecated.
  "Cython.Distutils.old_build_ext does not properly handle dependencies "
running test
running build
running build_py
copying regions/_compiler.c -> build/lib.macosx-10.12-x86_64-3.5/regions
copying regions/_geometry/circular_overlap.c -> build/lib.macosx-10.12-x86_64-3.5/regions/_geometry
copying regions/_geometry/core.c -> build/lib.macosx-10.12-x86_64-3.5/regions/_geometry
copying regions/_geometry/elliptical_overlap.c -> build/lib.macosx-10.12-x86_64-3.5/regions/_geometry
copying regions/_geometry/rectangular_overlap.c -> build/lib.macosx-10.12-x86_64-3.5/regions/_geometry
running build_ext
skipping 'regions/_geometry/circular_overlap.c' Cython extension (up-to-date)
skipping 'regions/_geometry/core.c' Cython extension (up-to-date)
skipping 'regions/_geometry/elliptical_overlap.c' Cython extension (up-to-date)
skipping 'regions/_geometry/rectangular_overlap.c' Cython extension (up-to-date)
usage: -c [options] [file_or_dir] [file_or_dir] [...]
-c: error: unrecognized arguments: --arraydiff
  inifile: /private/var/folders/sb/4qv5j4m90pz1rw7m70rj1b1r0000gn/T/regions-test-c0nb7we5/setup.cfg
  rootdir: /private/var/folders/sb/4qv5j4m90pz1rw7m70rj1b1r0000gn/T/regions-test-c0nb7we5
```

Eventually I did remember that `--arraydiff` is https://github.com/astrofrog/pytest-arraydiff that you added in #71, so I'll install that now.

My question is if you can make the error message better?

Instead of
```
-c: error: unrecognized arguments: --arraydiff
```
actually printing something like this would be good:
```
https://github.com/astrofrog/pytest-arraydiff not found.
Please `pip install pytest-arraydiff`
```

Also maybe mention and link to https://github.com/astrofrog/pytest-arraydiff here?
http://astropy-regions.readthedocs.io/en/latest/installation.html#optional-dependencies

**Comment by astrofrog** [comment]
I think the correct solution is to define pytest-arraydiff in ``tests_requires`` - I need to check if that can be added to ``setup.cfg`` though.

**Comment by astrofrog** [comment]
We need to merge https://github.com/astropy/package-template/pull/131 into the template first before we can do this - will try and get that moving.

**Comment by astrofrog** [comment]
Hmm, scrap that, ``tests_require`` doesn't work right at the moment, so that will need more work. In the mean time, check out https://github.com/astropy/regions/pull/80

**Comment by cdeil** [comment]
I think this issue was fully discussed and addressed in #80.
@astrofrog - Thanks!
